.circleci/config.yml

```diff
@@ -49,4 +47,4 @@ workflows:
             branches:
               ignore: /^.*/
             tags:
-              only: /^v.*/
+              only: /^v[0-9]+(\.[0-9]+)*(-.*)*/
```

Comment:
This is recommended by circleci to get the semver tags more precisely. Whereas before this would pick up `v1.0.0` and `vChrisIsAwesome`, the new regex requires me to use `v1.0.0-ChrisIsAwesome` before it will pick it up.

Reply:
this is a reegex that will. pattern match on any tag that looks like semver.